package.json

```diff
@@ -16,8 +16,6 @@
     "@mapbox/unitbezier": "^0.0.0",
     "brfs": "^1.4.0",
     "bubleify": "^0.5.1",
-    "csscolorparser": "^1.0.2",
-    "csscolorparser": "~1.0.2",
```

Comment:
Isn't that what [this](https://github.com/mapbox/mapbox-gl-js/blob/fac63584971bd9c1fcf076046e3c87e38509b5bc/package.json#L141) does?